src/main/java/com/github/flamingsharky/darkagesexpedition/world/DAEOreGen.java

```diff
@@ -0,0 +1,52 @@
+package com.github.flamingsharky.darkagesexpedition.world;
```

Comment:
Also move this one to `com.github.flamingsharky.darkagesexpedition.common.world`